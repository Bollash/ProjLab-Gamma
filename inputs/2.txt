Create Space
Add Asteroid Iron
Add Asteroid Iron
Add Settler 0
Add TpGate Asteroid 0 Asteroid 1
Move 0 0
Status All