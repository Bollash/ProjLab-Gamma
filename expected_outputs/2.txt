Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
Sikeres volt a telepeshozzáadás
Hozzáadtuk a teleportkapukat a célpontokhoz.
Az aktor az aszteroidára mozgott.
Space
1
10
10
False
2
3
Asteroid
0
3
3
3
Iron
1
tp 1
1
1
Asteroid
1
3
3
3
Iron
1
tp 2
2
0
2
Settler
0
1
0
0
TpGate
1
ast 0
2
False
TpGate
2
ast 1
1
False
